taxcalc/simpletaxio.py

```diff
@@ -0,0 +1,648 @@
+"""
+OSPC taxcalc package simple tax input-output class.
+"""
+# CODING-STYLE CHECKS:
+# pep8 --ignore=E402 simpletaxio.py
+# pylint --disable=locally-disabled simpletaxio.py
+
+import os
+import sys
+import json
+import re
+import numpy.core as np
```

Comment:
@talumbau,  You are right about this.  I've change to "import numpy as np" and "np.core.array(...)".  Need the .core to avoid pylint warning (seems that numpy has an 'unusual' importing philosophy that pylint has trouble with).  But working around this is now better than before, so thanks for the suggestion.